collections/ansible_collections/cloudkit/templates/roles/ocp_virt_vm/defaults/main.yaml

```
@@ -0,0 +1,33 @@
+---
+# Default VM configuration values
+
+# Default OS and Image Configuration
+default_vm_os_type: "linux"
+default_vm_image_source: "quay.io/containerdisks/fedora:latest"
+
+# Network Configuration Defaults
+default_vm_network_type: "pod"
+default_vm_expose_service: false
+default_vm_service_type: "ClusterIP"
+
+# Default service ports for common applications
+default_service_ports:
+  ssh:
+    - name: "ssh"
+      port: 22
+      target_port: 22
+      protocol: "TCP"
+
+# Storage Defaults
+default_storage_class: "nfs-client"  # NFS storage operator storage class
```

```suggestion
default_vm_storage_class: "nfs-client"  # NFS storage operator storage class
```

to make the naming consistent